test/parallel_api/algorithm/alg.modifying.operations/transform_unary_zip.pass.cpp

```diff
@@ -0,0 +1,115 @@
+// -*- C++ -*-
+//===-- transform_unary.pass.cpp ------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// Tests for transform
+
+#include "support/test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+#include _PSTL_TEST_HEADER(algorithm)
+
+#include "support/utils.h"
+
+using namespace TestUtils;
+
+template <typename... Types>
+using typle_t = oneapi::dpl::__internal::tuple<Types...>;
+//using typle_t = ::std::tuple<Types...>;
```

Comment:
@rarutyun May you take a look why I unable to use `td::tuple` in this test?
I have compile error in this case.